test/commands/bundle/bundle.test.ts

```diff
@@ -0,0 +1,104 @@
+import { test } from '@oclif/test';
+import fs from 'fs';
+import path from 'path';
+
+function cleanup(filepath: string) {
```

Comment:
looks like good candidate for reusable helper -> https://github.com/asyncapi/cli/blob/master/test/commands/generate/fromTemplate.test.ts#L12